client/rest-high-level/src/test/java/org/elasticsearch/client/CrudIT.java

```diff
@@ -265,4 +344,14 @@ public void testIndex() throws IOException {
                          "version conflict, document already exists (current version [1])]", exception.getMessage());
         }
     }
+
+    /**
+     * Index a document
+     * @param request index request
+     * @throws IOException if something goes wrong while executing the index request
+     */
+    private void createSampleDocument(IndexRequest request) throws IOException {
+        IndexResponse indexResponse = execute(request, highLevelClient()::index, highLevelClient()::indexAsync);
```

Comment:
one more thing, given that this is used only to test deletions, let's just use the sync version of the method rather than randomizing. I do wonder whether this additional method is needed.